docs/static/monitoring/monitoring-internal.asciidoc

```diff
@@ -89,54 +58,72 @@ can specify the location of the PEM encoded certificate with the
 
 Alternatively, you can configure trusted certificates using a truststore
 (a Java Keystore file that contains the certificates):
 
 [source,yaml]
 --------------------------------------------------
-xpack.monitoring.elasticsearch.ssl.truststore.path: /path/to/file
-xpack.monitoring.elasticsearch.ssl.truststore.password: password
+monitoring.elasticsearch.ssl.truststore.path: /path/to/file
+monitoring.elasticsearch.ssl.truststore.password: password
 --------------------------------------------------
 
 Also, optionally, you can set up client certificate using a keystore
 (a Java Keystore file that contains the certificate):
 
 [source,yaml]
 --------------------------------------------------
-xpack.monitoring.elasticsearch.ssl.keystore.path: /path/to/file
-xpack.monitoring.elasticsearch.ssl.keystore.password: password
+monitoring.elasticsearch.ssl.keystore.path: /path/to/file
+monitoring.elasticsearch.ssl.keystore.password: password
 --------------------------------------------------
 
 Set sniffing to `true` to enable discovery of other nodes of the {es} cluster.
 It defaults to `false`.
 
 [source,yaml]
 --------------------------------------------------
-xpack.monitoring.elasticsearch.sniffing: false
+monitoring.elasticsearch.sniffing: false
 --------------------------------------------------
 
 --
 
 . Restart your Logstash nodes.
 
-. To verify your {monitoring} configuration, point your web browser at your {kib}
+. To verify your monitoring configuration, point your web browser at your {kib}
 host, and select **Monitoring** from the side navigation. Metrics reported from
 your Logstash nodes should be visible in the Logstash section. When security is
-enabled, to view the monitoring dashboards you must log in to {kib} as a user
-who has the `kibana_user` and `monitoring_user` roles.
+enabled, you must log in to {kib} as a user who has the `kibana_user` and
+`monitoring_user` roles.
 +
 image::images/monitoring-ui.png["Monitoring",link="monitoring/images/monitoring-ui.png"]
 
-[float]
-[[monitoring-upgraded-logstash]]
-===== Re-enabling Logstash Monitoring After Upgrading
+include::../settings/monitoring-settings.asciidoc[]
 
-When upgrading from older versions of {xpack}, the built-in `logstash_system`
-user is disabled for security reasons. To resume monitoring,
-change the password and re-enable the logstash_system user.
 
-include::../settings/monitoring-settings.asciidoc[]
+[[internal-collector-components]]
+==== How {ls} monitoring with internal collectors works
+
+Monitoring {ls} with internal collectors uses these components:
+
+* <<logstash-monitoring-collectors,Collectors>>
+* <<logstash-monitoring-output,Output>>
 
+These pieces live outside of the default Logstash pipeline in a dedicated
+monitoring pipeline. This configuration ensures that all data and processing has
+a minimal impact on ordinary Logstash processing. Existing Logstash features,
+such as the <<plugins-outputs-elasticsearch,`elasticsearch` output>>, can be
```

Comment:
I know this comes from previously written copy - but I'm not sure what it is trying to say. I _think_ it is trying to talk about the reuse of certain features from the elasticsearch output, such as retry, but I'm not 100%